packages/flutter/test/material/floating_action_button_theme_test.dart

```diff
@@ -22,17 +22,25 @@ void main() {
       ),
     ));
 
+    final ElevatedButton elevatedButton = tester.widget(find.byType(ElevatedButton));
```

Comment:
This could even simplify things a bit:
```suggestion
    final ButtonStyle style = tester.widget(find.byType(ElevatedButton)).style!;
```